codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/alltypesrequest.java

```diff
@@ -1551,26 +1545,26 @@ public final void setStructWithNestedTimestampMember(StructWithTimestamp.Builder
         }
 
         public final ByteBuffer getBlobArg() {
-            return blobArg;
+            return blobArg == null ? null : blobArg.asByteBuffer();
         }
 
         @Override
-        public final Builder blobArg(ByteBuffer blobArg) {
+        public final Builder blobArg(SdkBytes blobArg) {
             this.blobArg = StandardMemberCopier.copy(blobArg);
             return this;
         }
 
         public final void setBlobArg(ByteBuffer blobArg) {
-            this.blobArg = StandardMemberCopier.copy(blobArg);
+            blobArg(blobArg == null ? null : SdkBytes.fromByteBuffer(blobArg));
         }
 
         public Builder blobArg(String blobArg) {
-            blobArg(new StringToByteBufferAdapter().adapt(blobArg));
+            blobArg(new StringToSdkBytesAdapter().adapt(blobArg));
             return this;
         }
 
         public void setBlobArg(String blobArg) {
-            this.blobArg = new StringToByteBufferAdapter().adapt(blobArg);
+            this.blobArg = new StringToSdkBytesAdapter().adapt(blobArg);
```

Comment:
should this just be `blobArg(blogArg)`?

Reply:
blobArg is a String and blobArg() takes an SdkBytes